main.js

```diff
@@ -1 +1,80 @@
-console.log('Hello world')
\ No newline at end of file
+{console.log('Hello world')
+let time, seconds, fullSec, fullMin, fullHour, clicked, turned;
+const frame = 60;
+
+const DOMS = {
+    startBtn: "#btn-start",
+    hours: "#hours",
+    minutes: "#minutes",
+    seconds: "#seconds",
+}
+
+const formatNumber = (number, time) =>
+{
+    if (typeof(number) === 'number' && !NaN) {
+        if (number < 10) {
+            document.querySelector(time).textContent = `0${number}`;
+        } else {
+            document.querySelector(time).textContent = `${number}`;
+        }

```

Comment:
Optional, but you could make this a turnery

```
document.querySelector(time).textContent = number < 10 ? `0${number}` : `${number}`
```

Or even

```
document.querySelector(time).textContent = `${number < 10 ? 0 : ''}${number}`
```

Whatever you think is most readable